lua/lspconfig.lua

```diff
@@ -4,71 +4,10 @@ local M = {
   util = require 'lspconfig/util';
 }
 
-function M.available_servers()
-  return vim.tbl_keys(configs)
-end
-
-function M.installable_servers()
-  local res = {}
-  for k, v in pairs(configs) do
-    if v.install then table.insert(res, k) end
-  end
-  return res
-end
-
 M._root = {}
--- Called from plugin/lspconfig.vim because it requires knowing that the last
--- script in scriptnames to be executed is lspconfig.
-function M._root._setup()
-  M._root.commands = {
-    LspInstall = {
```

Comment:
neovim 0.5 hasn't been released yet so in theory we could just get rid of this but aving the commands print a message like `print("deprecated, see https://github.com/neovim/neovim/wiki/Following-HEAD")  would avoid some pain for everyone (users and contributors on gitter). And we would remove it just before 0.5.

Reply:
Added in a separate commit to make it easy to remove 🔪